felix/bpf-gpl/ut/icmp_port_unreachable.c

```diff
@@ -9,7 +9,12 @@
 static CALI_BPF_INLINE int calico_unittest_entry (struct __sk_buff *skb)
 {
 	struct cali_tc_ctx ctx = {
+		.counters = counters_get(),
```

Comment:
you could have just provided a dummy on stack datablock, you would not need to do the checks below. It never calls any bpf map helper after this, right?

Reply:
It ends up calling `skb_refresh_validate_ptrs()` which calls `bpf_skb_pull_data()`. If we don't init `.counters` the object file will be rejected by the verifier.